test/options-handler.js

```diff
@@ -155,4 +173,19 @@ describe('Options handler', () => {
         });
         optionsHandler.setCompilers([]);
     });
+    it('should get static libraries', () => {
```

Comment:
this is great! Thanks :)

One note that confused me to start with: I think of "static link" as meaning `-static` but I don't know if that's what's happening here? We're linking with the default (which will be dynamic linking I think), right?

Reply:
This is more about linking to .a's into your binary, which doesn't really need supplying -static. But -static could be a good idea to play with as well.